### PR TITLE
Support bullseye build image

### PR DESCRIPTION
**Pull request by qianz2**
- [x] The purpose of this PR is explained in this message or in an issue. If an issue please include a reference as \#\<issue\_number\>.
- [x] Tests are included and/or updated for code changes.
- [x] Proper license headers are included in each file.

### CODE REVIEW

File: images/build/Dockerfiles/gitHubRunners.BuildPackDepsBullseye.Dockerfile [hunk 1]
```diff
@@ -0,0 +1,2 @@
+# https://github.com/actions/virtual-environments/blob/master/images/linux/Ubuntu1804-README.md
+FROM buildpack-deps:bullseye-scm
```
Comment (daniv-msft) [inline images/build/Dockerfiles/gitHubRunners.BuildPackDepsBullseye.Dockerfile:2]: Should we define the SHA, as we do for Stretch/Buster, to make sure we're not broken if Debian pushes a new update to the base image?

File: tests/Oryx.BuildImage.Tests/Php/PhpDynamicInstallationTest.cs [hunk 2]
```diff
@@ -44,9 +44,8 @@ public PhpDynamicInstallationTest(ITestOutputHelper output) : base(output)
                     PhpVersions.Composer23Version
                 );
                 data.Add(PhpVersions.Php74Version, imageHelper.GetGitHubActionsBuildImage(), PhpVersions.Composer23Version);
-                data.Add(PhpVersions.Php80Version, imageHelper.GetGitHubActionsBuildImage("buster"), PhpVersions.Composer23Version);
-                data.Add("8.1.4", imageHelper.GetGitHubActionsBuildImage("buster"), PhpVersions.Composer23Version);
-
+                data.Add(PhpVersions.Php80Version, imageHelper.GetGitHubActionsBuildImage("github-actions-buster"), PhpVersions.Composer23Version);
+                data.Add("8.1.4", imageHelper.GetGitHubActionsBuildImage("github-actions-buster"), PhpVersions.Composer23Version);
```
Comment (daniv-msft) [inline tests/Oryx.BuildImage.Tests/Php/PhpDynamicInstallationTest.cs:48]: Thanks for replacing `buster` by `github-actions-buster` and fixing this! I imagine the test was defaulting to stretch image before.

File: images/build/Dockerfiles/gitHubRunners.BuildPackDepsBullseye.Dockerfile (hunk 1 above)
Comment (qianz2) [reply images/build/Dockerfiles/gitHubRunners.BuildPackDepsBullseye.Dockerfile:2]: Thanks, that's good point, will update here soon once tested the sha locally

File: build/buildBuildImages.sh [hunk 3]
```diff
@@ -409,6 +418,7 @@ function buildBuildPackImage() {
 }
 
 if [ -z "$imageTypeToBuild" ]; then
+	buildGitHubActionsImage "bullseye"
```
Comment (cormacpayne) [inline build/buildBuildImages.sh:421]: Are we only creating a bullseye-based image for the GitHub Actions image right now? (_i.e._, ignoring Jamstack, LTS, etc.)

**Review by cormacpayne — DISMISSED** [review]

File: build/buildBuildImages.sh (hunk 3 above)
Comment (qianz2) [reply build/buildBuildImages.sh:421]: Thanks for bringing it up, for this PR we only supported the github-actions bullseye based image. Not sure if Jamstack or lts-version image are also needed from App Service, will follow up with them regarding it.

File: images/build/Dockerfiles/gitHubRunners.BuildPackDepsBullseye.Dockerfile (hunk 1 above)
Comment (daniv-msft) [reply images/build/Dockerfiles/gitHubRunners.BuildPackDepsBullseye.Dockerfile]: Thanks!

**Review by daniv-msft — APPROVED** [review]

File: build/buildBuildImages.sh (hunk 3 above)
Comment (cormacpayne) [reply build/buildBuildImages.sh:421]: Makes sense -- thanks for clarifying 😃

**Comment by qianz2** [comment]
Not sure why license/cla check was not initialized automatically after signing up CLA agreements.

**Comment by daniv-msft** [comment]
> Not sure why license/cla check was not initialized automatically after signing up CLA agreements.

Not sure either. Maybe we just need to trigger a pipeline? Not sure how to though. Or just push a new dummy commit to retrigger an udpate?

**Comment by qianz2** [comment]
> > Not sure why license/cla check was not initialized automatically after signing up CLA agreements.
> 
> Not sure either. Maybe we just need to trigger a pipeline? Not sure how to though. Or just push a new dummy commit to retrigger an udpate?

Ok, will try that, thanks

**Review by daniv-msft — APPROVED** [review]

**Pull request merged**